riscv/tests/riscv.rs

```diff
@@ -180,14 +180,6 @@ fn keccak() {
     verify_riscv_crate(case, Default::default(), false);
 }
 
-#[cfg(feature = "estark-polygon")]
-#[test]
-#[ignore = "Too slow"]
-fn vec_median_estark_polygon() {
-    let case = "vec_median";
-    verify_riscv_crate(case, &[5u64, 11, 15, 75, 6, 5, 1, 4, 7, 3, 2, 9, 2], true);
-}
```

Comment:
This test appears to be identical to `vec_median`!

Reply:
It is, but it was made a separate test because it runs in its own job, it's the only riscv test that's run with estark-polygon

Reply:
Meaning I dont think we should remove it

Reply:
OK, I undid it!

I'm not sure I understood this though, if they are the same, why can't we run the other one with the `estark-polygon` feature? Could be nice to add a comment about that as well.

Anyway, not the topic of this PR...